### PR TITLE
Feature #182 time service tests

### PR DESCRIPTION
**Pull request by bartosz-wojczynski**
Adds tests for AUR_CBI_MDL-0040, AUR_CBI_MDL-0050, AUR_CBI_MDL-0070

### CODE REVIEW

File: requirements/AUR_CBI_MDL-0040/work/environment/implem/default/C/Makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+VISUAL?=kate
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0040/work/environment/implem/default/C/Makefile]: Remove

File: requirements/AUR_CBI_MDL-0040/work/environment/implem/default/C/src/environment.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/* Header file for function Environment
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0040/work/environment/implem/default/C/src/environment.h]: Remove

File: requirements/AUR_CBI_MDL-0050/work/environment/implem/default/C/Makefile [hunk 3]
```diff
@@ -0,0 +1,14 @@
+VISUAL?=kate
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0050/work/environment/implem/default/C/Makefile]: Remove

File: requirements/AUR_CBI_MDL-0050/work/environment/implem/default/C/src/environment.h [hunk 4]
```diff
@@ -0,0 +1,43 @@
+/* Header file for function Environment
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0050/work/environment/implem/default/C/src/environment.h]: Remove

File: requirements/AUR_CBI_MDL-0070/work/environment/implem/default/C/Makefile [hunk 5]
```diff
@@ -0,0 +1,14 @@
+VISUAL?=kate
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0070/work/environment/implem/default/C/Makefile]: Remove

File: requirements/AUR_CBI_MDL-0070/work/environment/implem/default/C/src/environment.h [hunk 6]
```diff
@@ -0,0 +1,43 @@
+/* Header file for function Environment
```
Comment (Lurkerpas) [inline requirements/AUR_CBI_MDL-0070/work/environment/implem/default/C/src/environment.h]: Remove

File: requirements/AUR_CBI_MDL-0040/work/environment/implem/default/C/Makefile (hunk 1 above)
Comment (bartosz-wojczynski) [reply requirements/AUR_CBI_MDL-0040/work/environment/implem/default/C/Makefile]: Removed

File: requirements/AUR_CBI_MDL-0040/work/environment/implem/default/C/src/environment.h (hunk 2 above)
Comment (bartosz-wojczynski) [reply requirements/AUR_CBI_MDL-0040/work/environment/implem/default/C/src/environment.h]: Removed

File: requirements/AUR_CBI_MDL-0050/work/environment/implem/default/C/Makefile (hunk 3 above)
Comment (bartosz-wojczynski) [reply requirements/AUR_CBI_MDL-0050/work/environment/implem/default/C/Makefile]: Removed

File: requirements/AUR_CBI_MDL-0050/work/environment/implem/default/C/src/environment.h (hunk 4 above)
Comment (bartosz-wojczynski) [reply requirements/AUR_CBI_MDL-0050/work/environment/implem/default/C/src/environment.h]: Removed

File: requirements/AUR_CBI_MDL-0070/work/environment/implem/default/C/Makefile (hunk 5 above)
Comment (bartosz-wojczynski) [reply requirements/AUR_CBI_MDL-0070/work/environment/implem/default/C/Makefile]: Removed

File: requirements/AUR_CBI_MDL-0070/work/environment/implem/default/C/src/environment.h (hunk 6 above)
Comment (bartosz-wojczynski) [reply requirements/AUR_CBI_MDL-0070/work/environment/implem/default/C/src/environment.h]: Removed

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**